harness/determined/common/api/bindings.py

```diff
@@ -86,16 +86,29 @@ def __str__(self) -> str:
 
 
 class GetMasterResponseProduct(DetEnum):
+    """Different kinds of Determined Cloud offerings
```

Comment:
it'd be nice if these followed our python style guide for docstrings, something like:
```
    """Short description of the class.

    Attributes:
        ENUM1: description
        ENUM2: description
        ENUM3: description
    """
```

Reply:
the first line wants a short description but I don't have a good way of programmatically and reliably figuring out what's a summary and what's not.

regarding how the attributes are listed I first tried the format you described and I found it it didn't show up as nicely in my IDE (VIM) and added more lines so I switched it but it makes sense to stick to the same style guide for these as well. the upside is that since these are generated switching them back and forth doesn't have a high cost.

Reply:
sure, sounds good.